### PR TITLE
mobile: i18n

### PR DESCRIPTION
**Pull request by kayleegeorge**
- add i18n

### CODE REVIEW

File: apps/daimo-mobile/src/i18n/languages/en.ts [hunk 1]
```diff
@@ -444,10 +446,13 @@ export const en = {
   // DepositScreen.tsx
   deposit: {
     screenHeader: () => `Deposit or Withdraw`,
+    go: () => `Go`,
```
Comment (kayleegeorge) [inline apps/daimo-mobile/src/i18n/languages/en.ts:449]: @Ethanol48 caught these newly added Landline text -- could you add to Spanish? Thank you so much!

File: apps/daimo-mobile/src/view/screen/SettingsScreen.tsx [hunk 2]
```diff
@@ -96,21 +110,60 @@ function AccountSection({ account }: { account: Account }) {
...
+        paddingHorizontal: 24,
+        paddingVertical: 13,
+        flexDirection: "row",
+        justifyContent: "space-between",
+      }}
+    >
+      <DaimoText variant="dropdown">{localeToLanguage(locale)}</DaimoText>
+    </View>
+  );
+}
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/screen/SettingsScreen.tsx:165]: can set in system settings

**Review by dcposch — APPROVED** [review]
🚀

File: apps/daimo-mobile/src/i18n/languages/en.ts [hunk 3]
```diff
@@ -0,0 +1,783 @@
+import { LanguageDefinition } from "../generate/languageDefinition";
+
+export const en: LanguageDefinition = {
```
Comment (kayleegeorge) [inline apps/daimo-mobile/src/i18n/languages/en.ts:3]: @Ethanol48 Added a type LanguageDefinition -- should should you errors if you are missing anything

**Pull request merged**